docs/Learning-Environment-Create-New.md

```diff
@@ -105,31 +85,36 @@ component later in the tutorial.
 To create the Agent:
 
 1. Select the RollerAgent GameObject to view it in the Inspector window.
-2. Click **Add Component**.
-3. Click **New Script** in the list of components (at the bottom).
-4. Name the script "RollerAgent".
-5. Click **Create and Add**.
+1. Click **Add Component**.
+1. Click **New Script** in the list of components (at the bottom).
+1. Name the script "RollerAgent".
+1. Click **Create and Add**.
 
 Then, edit the new `RollerAgent` script:
 
 1. In the Unity Project window, double-click the `RollerAgent` script to open it
-   in your code editor.
-2. In the editor, add the `using MLAgents;` and `using MLAgents.Sensors` statements and then change the base
-    class from `MonoBehaviour` to `Agent`.
-3. Delete the `Update()` method, but we will use the `Start()` function, so
-    leave it alone for now.
+in your code editor.
+1. In the editor, add the `using MLAgents;` and `using MLAgents.Sensors` statements and then change the base class from `MonoBehaviour` to `Agent`.
+1. Delete the `Update()` method, but we will use the `Start()` function, so
+leave it alone for now.
 
 So far, these are the basic steps that you would use to add ML-Agents to any
 Unity project. Next, we will add the logic that will let our Agent learn to roll
-to the cube using reinforcement learning.
+to the cube using reinforcement learning. More specifically, we will need to extend
+three methods from the `Agent` base class:
+* `OnEpisodeBegin()`
+* `CollectObservations(VectorSensor sensor)`
+* `OnActionReceived(float[] vectorAction)`
+
+We overview each of these in more detail in the dedicated subsections below.
 
 ### Initialization and Resetting the Agent
 
-When the Agent reaches its target, its episode ends and the `OnEpisodeBegin()`
-method moves the target to a random location. In addition, if the Agent rolls
+When the Agent (Sphere) reaches its target (Cube), its episode ends and the `OnEpisodeBegin()`
```

Comment:
This paragraph doesn't actually talk about "Initialization" even though it's in the section title. Maybe mention that `Initialization` is called at the start of simulation too?

Reply:
Good catch.